content/data-types/exercises/_index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "Exercises"
+date: 2023-01-17T16:28:22-06:00
+draft: false
+weight: 2
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+To get started, open up Visual Studio and create a new console program.
+
+<!-- TODO: Add link to intro -->
+
+{{% notice green "Tip" "rocket" %}}
+
+   If you do not recall how to create a new console program or need a quick refresher to get started, check out the section on creating a C# project.
+
+{{% /notice %}}
+
+For each part of the exercises, create a new project in your solution. When you are done, push your solution up to a repository on your Github account!
+
+## Input/Output
+
+1. Write a new "Hello, World" program to prompt the
+   user for their name and greet them by name.
+
+   a. Add a question to ask the user:
+
+      ```csharp
+         Console.WriteLine("What is your name?");
+      ```
+
+   b. Create a variable to store the user's input:
+
+      ```csharp
+         string name = Console.ReadLine(); 
+      ```
+
+   c. Use concatenation to print the greeting:
+
+      ```csharp
+         Console.WriteLine("Hello " + name);
+      ```
+
+   d. Run your program.
+
+{{% expand "Click Here for Answer" %}}
+
+   ```csharp {linenos = table}
+      Console.WriteLine("What is your name?");
+      string myName = Console.ReadLine();
+      Console.WriteLine("Hello " + myName + "!");
+   ```
+
+{{% /expand %}}
```

Comment:
```suggestion
  {{% expand "Check your solution" %}}
  
     ```csharp {linenos = table}
        Console.WriteLine("What is your name?");
        string myName = Console.ReadLine();
        Console.WriteLine("Hello " + myName + "!");
     ```
  
  {{% /expand %}}
```

Reply:
Done!

Reply:
Thanks